src/components/v2v3/V2V3Project/ProjectSafeDashboard/SafeTransaction.tsx

```diff
@@ -16,66 +18,82 @@ const nonceStyle: CSSProperties = {
   width: '1rem',
 }
 
-function TransactionHeader({
+export const safeTransactionRowStyle: CSSProperties = {
+  display: 'flex',
+  justifyContent: 'space-between',
+  fontWeight: 400,
+  width: '100%',
+  padding: '0.5rem 1rem',
+  marginBottom: '1rem',
+  transition: 'background-color 100ms linear',
+}
+
+export function TransactionHeader({
   transaction,
+  onClick,
+  title,
 }: {
   transaction: SafeTransactionType
+  onClick?: VoidFunction
+  title?: string
 }) {
   const {
     theme: { colors },
   } = useContext(ThemeContext)
+  const _method = title ?? transaction?.dataDecoded?.method
+  const transactionTitle = (
+    <Tooltip title={t`Go to Safe`}>
```

Comment:
**suggestion** probably wouldn't even both with the tooltip. Its self-explanatory with the hover state imo

Reply:
@tomquirk true ok